AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -369,9 +384,22 @@ - (void)trackPage:(NSString *)pageName withProperties:(NSDictionary<NSString *,
 }
 
 - (void)sendLog:(id<MSLog>)log flags:(MSFlags)flags {
+  [(flags & MSFlagsCritical) != 0 ? self.criticalChannelUnit : self.channelUnit enqueueItem:log flags:flags];
+}
 
-  // Send log to log manager.
-  [self.channelUnit enqueueItem:log flags:flags];
+- (void)setTransmissionInterval:(NSUInteger)interval {
+  if (self.started) {
+    MSLogError([MSAnalytics logTag], @"The transmission interval should be set before the MSAnalytics service is started.");
+    return;
+  }
+  if (interval > kMSFlushIntervalMaximum || interval < kMSFlushIntervalMinimum) {
+    MSLogError(
+        [MSAnalytics logTag], @"The transmission interval is not valid, it should be between %u second(s) and %u second(s) (%u day(s)).",
```

Comment:
Please take a look at [this](https://github.com/microsoft/appcenter-sdk-android/pull/1158#discussion_r284368478). 
We just need to say the interval is not valid.
```
    MSLogError([MSAnalytics logTag], @"The transmission interval is not valid.");
```